### ISSUE TITLE
MyMonero: Can`t send XMR - Error: Count cannot be 11

### ISSUE THREAD

**Issue by Thein91** [mymonero/mymonero-app-js#507]
Hey to all, 

for me it is impossible to send XMR. I have already searched the internet for that problem, unfortunately without success. There is always that error coming up: 

![Error_MMonero](https://user-images.githubusercontent.com/111884240/186173010-1b4ef1c1-2e62-49aa-b0ab-f76c2d448db9.jpg)
 
I use the latest version of MyMonero (1.3) on a windows 10 64-bit. 
Thanks for any help! 
Kind regards,
T

**Comment by mikakun** [comment]
same issue with the .appimage

**Comment by mikakun** [comment]
i have millions & millions of block to sync to restore my wallets somewhere else (feather) & a limited internet connection 😭😭😭😭😭😭😭😭😭😭

**Comment by franktheta** [comment]
yeah , unfortunately i have the same problem . Is there anyone who can help ?

**Comment by franktheta** [comment]
since when do you have this problem? 2 weeks ago (and long before that) it worked without any problems.

Is there any chance to talk with an admin or someone else , pls

**Comment by fdterr** [comment]
+1 also having this issue, what is going on here?

**Comment by franktheta** [comment]
It seems everyone has this problem . The question is ... wtf we can do ??? :/

**Comment by franktheta** [comment]
I agree with you! That's actually unacceptable. I wrote to the support (MYmonero) and hope that they will answer me. Dieser Scheiß muss echt nicht sein !

**Comment by trasherdk** [comment]
Easy fix. Install [Monero GUI Wallet](https://www.getmonero.org/downloads/#gui) and restore from your seed.

**Comment by hack-r** [comment]
This happens to me on both iOS and Linux.

**Comment by hack-r** [comment]
> Easy fix. Install [Monero GUI Wallet](https://www.getmonero.org/downloads/#gui) and restore from your seed.

Did you try that successfully or just speculating? I tried restoring the MyMonero wallet to Electrum but it failed. The MyMonero seed is not in the expected format; I guessed this mean I needed to specify BLIP. Doing so technically restored the wallet but the balance didn't show up.

**Comment by trasherdk** [comment]
It has been done successfully by a number of people, reported on IRC/Matrix.

I don't use `mymonero` myself, as I don't consider it a secure, or private, application.

**Comment by mikakun** [comment]
> > Easy fix. Install [Monero GUI Wallet](https://www.getmonero.org/downloads/#gui) and restore from your seed.
> 
> Did you try that successfully or just speculating? I tried restoring the MyMonero wallet to Electrum but it failed. The MyMonero seed is not in the expected format; I guessed this mean I needed to specify BIP-32. Doing so technically restored the wallet but the balance didn't show up.

restore with the keys in monero GUi (or feather) not the seed, you can get them from mymonero. If you have an old wallet & limited internet connection that's an issue cause it will take forever & burn bandwidth like mad, otherwise you're fine.

**Comment by KittyKattyKat** [comment]
Same issue...

**Comment by AslanPonies** [comment]
Hi @fdterr @mikakun @trasherdk @hack-r @Thein91 @KittyKattyKat @franktheta,

MyMonero are currently working on the latest Monero hardfork patch. Whilst we are scanning wallets to the current block height where you are able to view your xmr, we are working on the decoy selection issue which relates to send transactions not confirming. We will update you on when the spending function will be enabled. Will keep you updated with progress here.

**Comment by hack-r** [comment]
> It has been done successfully by a number of people, reported on IRC/Matrix.
> 
> 
> 
> I don't use `mymonero` myself, as I don't consider it a secure, or private, application.

So you comment on GitHub issues of software you don't use with solutions you haven't tried (as if they were facts)?!

This is **not** a good practice. Please don't.

**Comment by franktheta** [comment]
@AslanPonies  
That would be great, thank you ! Another question // Do you have a approximate idea / information when it will work again?

Maybe I'm wrong, but it seems that you have reliable contacts with the provider.

**Comment by KennethWussmann** [comment]
@franktheta The latest release fixes the issue, but only on Linux and macOS: https://github.com/mymonero/mymonero-app-js/releases/tag/v1.3.2

**Comment by devinpearson** [comment]
@KennethWussmann thanks Kenneth that's correct. Im just sorting out the ssl cert for the windows version. hoping to resolve that tomorrow

**Comment by hack-r** [comment]
iOS?On Aug 25, 2022, at 3:23 PM, Devin ***@***.***> wrote:﻿
@KennethWussmann thanks Kenneth that's correct. Im just sorting out the ssl cert for the windows version. hoping to resolve that tomorrow

**Comment by devinpearson** [comment]
@hack-r  iOS has been submitted to the AppStore for review

**Comment by LivBoss** [comment]
Here is an official announcement from MyMonero:

We are happy to announce that the MyMonero wallet is fully operational again.

We have successfully upgraded the MyMonero software to fall in line with the Monero hard fork.

This is true for all our web-wallet, MacOS and Linux users, as well as those operating on the Android and iOS mobile apps.

You will need to update to the latest software in order to fully operate on the new software.
If you are unsure about this, you can refer to our website - mymonero.com for the latest software versions.

You can now view your XMR balances and transactions, and you can also send funds again.

An important note to our Windows users:
We have successfully completed the hard fork upgrades on the Windows software and although we are very eager to release it, we are awaiting a certificate that will allow us to release this software publicly.
This process is a priority for us and we will announce once our Windows platforms are available again.

Feel free to reach out to us if you have any questions or concerns.

Thank you for your patience.

**Comment by guzzijones** [comment]
Is there a way to see my mnemonic seed?  I cannot complete a send from my wallet.

**Comment by LivBoss** [comment]
> Is there a way to see my mnemonic seed? I cannot complete a send from my wallet.

Please contact MyMonero support via the website, mymonero.com. Or via email at support@mymonero.com.
